spec/views/rb_taskboards/show_spec.rb

```diff
@@ -127,7 +135,7 @@
 
       stories.each do |story|
         expect(rendered).to have_selector "#story_#{story.id}" do
-          with_selector ".status", story.status.name
+          with_selector '.status', story.status.name
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.